qutip/floquet.py

```diff
@@ -702,20 +699,14 @@ def floquet_master_equation_tensor(Alist, f_energies):
         for J in range(N * N):
             c, d = vec2mat_index(N, J)
 
-            R = -1.0j * (f_energies[a] - f_energies[b])*(a == c)*(b == d)
```

Comment:
I think I might be missing something here, but I can't find where it comes from, while looking at eqs.(251-252) from the [Gri98] reference.

Reply:
Actually, it looks like it was [added a few years ago](https://github.com/qutip/qutip/commit/6942deb25cee0114a7ac0c9f9098dfd40006cbbe).

@jrjohansson would you have more infos about it please? :)